packages/cardano-services/src/Asset/CardanoTokenRegistry.ts

```diff
@@ -26,12 +26,10 @@ interface TokenMetadataServiceRecord {
   url?: StringValue;
 }
 
-const propertiesToChange: Record<string, string> = { description: 'desc', logo: 'icon' };
+const propertiesToChange: Record<string, string> = { description: 'desc', logo: 'icon', subject: 'assetId' };
 export const toCoreTokenMetadata = (record: TokenMetadataServiceRecord) =>
   Object.fromEntries(
-    Object.entries(record)
-      .filter(([key]) => ['decimals', 'description', 'logo', 'name', 'ticker', 'url'].includes(key))
```

Comment:
Why this `filter` was removed? Can we be 100% sure the source object has no extra attributes?

Reply:
It seems to have been filtered before this since the `assetId` [wasn't part of the return type](https://github.com/input-output-hk/cardano-js-sdk/pull/691/files#diff-309fb53fafaedbbbc947ab39a769ef9b9d72e50a0de381866a0b79020e33501aR25)